randovania/data/json_data/prime1.txt

```diff
@@ -4765,7 +4765,9 @@ Asset id: 413884678
   > Dock to Gathering Hall Access
       Trivial
   > Event - Open gate
-      Scan Visor and Morph Ball Bomb and Morph Ball and Bomb Jump (Intermediate)
+      Any of the following:
+          Scan Visor
+          Morph Ball Bomb and Morph Ball and Bomb Jump (Intermediate)
```

Comment:
I'm thinking that this line should be a path from the Front of Gate to Sunchamber lobby because you can't actually open the gate without scan visor.

Reply:
That makes more sense. Fixed